### PR TITLE
Add ciphersuite field to `PublicGroupStateTBS`

### LINKED ISSUE #498: `Ciphersuite` should be part of `PublicGroupStateTbs`
Am I missing something here, or should the ciphersuite really be part of the `PublicGroupStateTbs`? It's part of the `PublicGroupState`, but the way it is right now it doesn't get signed.

### PR DESCRIPTION
**Pull request by kkohbrok**
Fixes #498.

### CODE REVIEW

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -2965,6 +2965,7 @@ following structure, comprising all the fields in the PublicGroupState above
 
 ~~~~~
 struct {
+    CipherSuite cipher_suite;
```
Comment (raphaelrobert) [inline draft-ietf-mls-protocol.md:2968]:
```suggestion
    ProtocolVersion version = mls10;
    CipherSuite cipher_suite;
```

**Review by raphaelrobert — CHANGES_REQUESTED** [review]
Good point! The protocol version was also missing. New joiners learn this from the `Welcome` message normally, that's why it wasn't included in the `GroupInfo` again.

The protocol version should also be repeated in the `PublicGroupState`.

File: draft-ietf-mls-protocol.md (hunk 1 above)
Comment (kkohbrok) [reply draft-ietf-mls-protocol.md:2968]: Can't the ProtocolVersion be inferred from the Ciphersuite?

Comment (bifurcation) [reply draft-ietf-mls-protocol.md:2968]: No.  That's why they're separate fields in KeyPackage.

**Review by bifurcation — APPROVED** [review]

**Pull request merged**